contributing/giving-constructive-criticism.rst

```diff
@@ -0,0 +1,154 @@
+Giving constructive criticism
+=============================
+
+:doc:`Reviewing issues and pull request </contributing/community/reviews>` is a great way to get started with
+contributing to the Symfony community. Anyone can do it! But before
+you give a comment, take a step back and think, is what you are about
+to say actually what you indent?
+
+Communicating over the internet with nothing but text can pose a
+big challenge, especially if you remember the Symfony community
```

Comment:
I think that "...if you remember that the Symfony..." would be better here. I'm not sure, but in my head it sounds like there's something missing now.

Reply:
I had that originally but I try to avoid using "that" all the time 😃 maybe a more native speaker can give a suggestion for this.